### ISSUE TITLE
Block Category 3 regions for Brave Premium services

### ISSUE THREAD

**Issue by jamesmudgett** [brave/brave-browser#41676]
Block category 3 regions for both Brave Premium and Brave Rewards. Right now Brave Premium and Brave Rewards are blocked for category 2 regions, and Brave Rewards are blocked for category 3 regions. The goal here is to have category 3 regions (a superset of category 2) to have the same blocking as category 2

**Comment by mrose17** [comment]
I think the title should have s/in/and/

**Comment by kylehickinson** [comment]
Partial dupe: https://github.com/brave/brave-browser/issues/40830

**Comment by mrose17** [comment]
Good catch! I suggest we deal with both issues at the same time, as it's effectively the same logic.

**Comment by mrose17** [comment]
Can we also change the description to ```Block category 3 regions for both Brave Premium and Brave Rewards. Right now Brave Premium and Brave Rewards are blocked for category 2 regions, and Brave Rewards are blocked for category 3 regions. The goal here is to have category 3 regions (a superset of category 2) to have the same blocking as category 2.```
